catalystwan/models/configuration/feature_profile/sdwan/policy_object/security/ssl_decryption.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2024 Cisco Systems, Inc. and its affiliates
+
+from typing import Literal
+
+from pydantic import AliasPath, BaseModel, ConfigDict, Field
+
+from catalystwan.api.configuration_groups.parcel import Global, _ParcelBase
+
+Action = Literal["decrypt", "drop"]
+CertificateRevocationStatus = Literal["oscp", "none"]
+FailureMode = Literal["close", "open"]
+KeyModulus = Literal["1024", "2048", "4096"]
+EckeyType = Literal["P256", "P384", "P521"]
+TlsVersion = Literal["TLSv1", "TLSv1.1", "TLSv1.2"]
+CaTpLabel = Literal["PROXY-SIGNING-CA"]
+
+
+class CaCertBundle(BaseModel):
+    model_config = ConfigDict(populate_by_name=True)
```

Comment:
```suggestion
    model_config = ConfigDict(populate_by_name=True, extra="forbid")
```
Because it defaults to "ignore"